core.js

```diff
@@ -73,7 +73,15 @@ module.exports = function (xhr) {
       if (options.data && type === 'GET') {
           // make sure we've got a '?'
           options.url += includes(options.url, '?') ? '&' : '?';
-          options.url += qs.stringify(options.data);
+          // set stringify encoding options and create a different URI output if needed
+          // ex) qsOptions = { indices: false }
+          // https://www.npmjs.com/package/qs/v/4.0.0#stringifying
+          if(options.qsOptions){
```

Comment:
qs.stringify does a truthiness check for options to decide whether or not to parse that parameter, this if statement isn't even needed.

Reply:
Sure, I can remove the if statement since is not needed.

I tested the `qs.stringify(options.data, undefined)` and `qs.stringify(options.data, null)` and it works just fine.

Reply:
I remove the if statement.
I tested the `qs.stringify(options.data, undefined)` and `qs.stringify(options.data, null)` and it works just fine.